STARTERKIT/gulpfile.js

```diff
@@ -131,60 +130,42 @@ options.sprites = {
 // Use `options.gulpWatchOptions = {interval: 1000, mode: 'poll'};` as example.
 options.gulpWatchOptions = {};
 
-// Load Gulp and tools we will use.
-// Task gulp-load-plugins will report "undefined" error unless you load
-// gulp-sass manually.
-var gulp = require('gulp'),
-  $ = require('gulp-load-plugins')(),
-  browserSync = require('browser-sync').create(),
-  del = require('del'),
-  sass = require('gulp-sass'),
-  kss = require('kss'),
-  cache = require('gulp-cached'),
-  spritesmith = require('gulp.spritesmith');
+// Browsersync options
+options.browserSync = {
```

Comment:
Maybe better to have this one in a separated config file like we do on webpack?

Reply:
yep. in future plans. 
https://github.com/skilld-labs/zen/issues/44
also we need to move config.